appointment/reschedule/index.php

```diff
@@ -0,0 +1,125 @@
+<link rel="stylesheet" href="/dist/appointment/reschedule/clientReschedule.css">
+<!-- TemplateEndEditable -->
+<!-- TemplateParam name="class" type="text" value="" -->
+</head>
+<body class="hide-wdn_identity_management" data-version="4.1">
+	<?php wdnInclude("/wdn/templates_4.1/includes/skipnav.html"); ?>
+	<div id="wdn_wrapper">
+		<input type="checkbox" id="wdn_menu_toggle" value="Show navigation menu" class="wdn-content-slide wdn-input-driver" />
+		<?php wdnInclude("/wdn/templates_4.1/includes/noscript-padding.html"); ?>
+		<header id="header" role="banner" class="wdn-content-slide wdn-band">
+			<div id="wdn_header_top">
+				<span id="wdn_institution_title"><a href="http://www.unl.edu/">University of Nebraska&ndash;Lincoln</a></span>
+				<div id="wdn_resources">
+					<?php wdnInclude("/wdn/templates_4.1/includes/wdnResources.html"); ?>
+					<?php wdnInclude("/wdn/templates_4.1/includes/idm.html"); ?>
+					<?php wdnInclude("/wdn/templates_4.1/includes/search.html"); ?>
+				</div>
+			</div>
+			<div id="wdn_logo_lockup">
+				<div class="wdn-inner-wrapper">
+					<?php wdnInclude("/wdn/templates_4.1/includes/logo.html"); ?>
+						<span id="wdn_site_affiliation"><!-- TemplateBeginEditable name="affiliation" --><!-- TemplateEndEditable --></span>
+						<span id="wdn_site_title"><!-- TemplateBeginEditable name="titlegraphic" -->VITA Lincoln<!-- TemplateEndEditable --></span>
+				</div>
+			</div>
+		</header>
+		<div id="wdn_navigation_bar" class="wdn-band">
+			<nav id="breadcrumbs" class="wdn-inner-wrapper" role="navigation" aria-label="breadcrumbs">
+				<!-- TemplateBeginEditable name="breadcrumbs" -->
+				<ul>
+					<li><a href="http://www.unl.edu/" title="University of Nebraska&ndash;Lincoln" class="wdn-icon-home">UNL</a></li>
+					<li><a href="/" title="VITA Lincoln">VITA Lincoln</a></li>
+					<li>Cancel or Reschedule Appointment</li>
+				</ul>
+				<!-- TemplateEndEditable -->
+			</nav>
+			<div id="wdn_navigation_wrapper">
+				<nav id="navigation" role="navigation" aria-label="main navigation">
+					<!-- TemplateBeginEditable name="navlinks" -->
+					<?php include "$root/sharedcode/navigation.php"; ?>
+					<!-- TemplateEndEditable -->
+					<?php wdnInclude("/wdn/templates_4.1/includes/navigation-addons.html"); ?>
+				</nav>
+			</div>
+		</div>
+		<div class="wdn-menu-trigger wdn-content-slide">
+			<label for="wdn_menu_toggle" class="wdn-icon-menu">Menu</label>
+			<?php wdnInclude("/wdn/templates_4.1/includes/share.html"); ?>
+		</div>
+		<main id="wdn_content_wrapper" role="main" class="wdn-content-slide" tabindex="-1">
+			<div id="maincontent" class="wdn-main">
+				<div id="pagetitle">
+					<!-- TemplateBeginEditable name="pagetitle" -->
+					<h1>Cancel or Reschedule Appointment</h1>
+					<!-- TemplateEndEditable -->
+				</div>
+				<!-- TemplateBeginEditable name="maincontentarea" -->
+				<div id="clientRescheduleApp" class="wdn-band" client-reschedule token="<?php echo $token; ?>"></div>
```

Comment:
This is the way I'm passing the token value to the client-reschedule app, not sure if this is the best approach, but I couldn't just get the `$_REQUEST` in the appointments.php, I could only get it in the index.php for some reason, so I opted to pass the token as a parameter to our angular module. @CollinsSpencer thoughts?